README.md

```diff
@@ -131,6 +131,17 @@ __Note:__ It is not possible to install and uninstall an instance in the same ru
 When attempting to remove the default instance this can be only done when having the ``::nodejs`` class __NOT__ defined as otherwise ``duplicate resource`` errors would occur. 
 After that no new default instance will be configured.
 
+### Setup using custom amount of cpu cores
+
+By default, all available cpu cores are being used to compile nodejs. Set `use_cpus` to any number of cores you want to use.
+
+```puppet
+class { 'nodejs':
+  version      => 'stable',
+  use_cpus     => 2,
```

Comment:
please use appropriate alignment here (arrows should be one space longer than the longest parameter)